oam/install_scripts/columnstore-pre-uninstall.in

```diff
@@ -22,29 +21,32 @@ else
     kill $(pidof $PROGS) > /dev/null
     sleep 3
     kill -9 $(pidof $PROGS) > /dev/null
+    clearShm  > /dev/null 2>&1
```

Comment:
clearShm would have already been called during systemctl stop so only need this in here w/o systemd